cni-plugin/pkg/dataplane/linux/dataplane_linux.go

```diff
@@ -67,7 +68,7 @@ func (d *linuxDataplane) DoNetworking(
 
 	d.logger.Infof("Setting the host side veth name to %s", hostVethName)
 
-	hostNlHandle, err := netlink.NewHandle()
+	hostNlHandle, err := netlink.NewHandle(unix.NETLINK_ROUTE)
```

Comment:
Do `unix.NETLINK_ROUTE` is only related to route property?

Reply:
No, not only routing.

From the [netlink man page](https://www.man7.org/linux/man-pages/man7/netlink.7.html):
> NETLINK_ROUTE
>               Receives routing and link updates and may be used to
>               modify the routing tables (both IPv4 and IPv6), IP
>               addresses, link parameters, neighbor setups, queueing
>               disciplines, traffic classes, and packet classifiers (see
>               [rtnetlink(7)](https://www.man7.org/linux/man-pages/man7/rtnetlink.7.html)).

Reply:
I got it! thanks for your details.